company.rb

```diff
@@ -0,0 +1,29 @@
+require './employees'
+require './departments'
+
+mike = Employee.new("Mike", 100000)
+john = Employee.new("John", 50000)
+management = Department.new("Management")
+management.add(mike)
+management.add(john)
+
+john.satisfactory = false
+
+#puts management.dept_raise
```

Comment:
Best practice is to not keep commented-out code.  Ideally, you delete all code that's commented out before committing, and then you can always get back old code if you need it from the git history.